src/rules/objectShorthandRule.ts

```diff
@@ -0,0 +1,42 @@
+import * as Lint from "../lint";
+import * as ts from "typescript";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        ruleName: "object-shorthand",
+        description: "Enforces use of ES6 object literal shorthand when possible.",
+        options: null,
+        optionExamples: ["true"],
+        type: "style",
+    };
+    /* tslint:enable:object-literal-sort-keys */
+
+    public static LONGHAND_PROPERTY = "Expected property shorthand.";
+    public static LONGHAND_METHOD = "Expected method shorthand.";
```

Comment:
can we make the error messages more verbose too, like so?

- "Expected property shorthand in object literal."
- "Expected method shorthand in object literal."